src/container/srv_target.c

```diff
@@ -1154,6 +1166,14 @@ cont_child_destroy_one(void *vin)
 				       &cont);
 		if (rc == -DER_NONEXIST)
 			break;
+
+		if (cont->sc_stopping) {
+			rc = -DER_SHUTDOWN;
+			D_ERROR(DF_CONT": container is in stopping "DF_RC"\n",
+				DP_CONT(cont->sc_pool->spc_uuid, cont->sc_uuid), DP_RC(rc));
+			cont_child_put(tls->dt_cont_cache, cont);
+		}
```

Comment:
This looks incorrect to me, why should container destroy return error when it's stopping?

Reply:
Yup, this should be removed.